File: julia/raindrops/raindrops.jl
function raindrops(number::Int)
    PAIRS = [
        (3, "Pling"),
        (5, "Plang"),
        (7, "Plong"),
    ]
    map(t -> number % t[1] == 0 ? t[2] : "" , PAIRS) |> join |> str -> maximum(["$number", str])
end

File: python/raindrops/raindrops.py
def convert(number):
  ans = ""
  if number % 3 == 0:
    ans += "Pling"
  if number % 5 == 0:
    ans += "Plang"
  if number % 7 == 0:
    ans += "Plong"

  return str(number) if len(ans) == 0 else ans
